packages/app/src/components/pages/VerifyEmail/index.js

```diff
@@ -1,11 +1,19 @@
-import React from 'react'
+import React, { useEffect } from 'react'
+import utilsAPI from '../../../utils/API/nominationsAPI.js';
+
+const VerifyEmail = (props) => {
+  
+    const token = props.match.params.token
+
+    useEffect(()=> {
+        utilsAPI.validateEmail(token)
+    });
 
-const VerifyEmail = () => {
     return (
         <div>
-            <h2>Thank you for verifying your email!</h2>
+            <h4>Thank you for verifying your email!</h4>
         </div>
     )
-}
+}    
 
-export default VerifyEmail
+export default VerifyEmail;
```

Comment:
Nitpick: It's usually good from to end files with a newline.

When you don't, GitHub throws up this little ⛔️ icon.

Reply:
interesting, didn't know that, got it